### PR TITLE
Updates to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Cadair**
More links to places and make it a little more friendly.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,49 @@
 How to contrubute to SunPy
 --------------------------
 
+Hello, and thanks for being interested in contributing to SunPy!
```
Comment (wafels) [inline CONTRIBUTING.md]: Thanks for your interest in contributing to SunPy!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,49 @@
...
+
 SunPy is an open project that enourages everyone to contribute in any way possible.
-This document descibes how to contribute code and documentation to the main SunPy 
-repository, if you wish to make a different contribution please contact the 
-[mailing list](https://groups.google.com/forum/#!forum/sunpy).
+This document mainly focuses on code or documentation contributions, however
```
Comment (wafels) [inline CONTRIBUTING.md]: Although this document mainly focuses on how to make contributions to SunPy code and documentation, there are many other ways to get involved with the SunPy community.  A good place to start is by saying hello on  the [mailinglist]() or [IRC]().

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,20 +1,49 @@
 How to contrubute to SunPy
```
Comment (wafels) [inline CONTRIBUTING.md]: How to contribute to SunPy

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,20 +1,49 @@
...
+This document mainly focuses on code or documentation contributions, however
+there are many other ways to get involved with the SunPy community, a good place
+to start is saying hello on the [mailinglist]() or [IRC]().
+
+
+## Reporting Issues
+
+If you have found a problem when using SunPy the best way to report it and get
+some help is by opening an [issue](https://github.com/sunpy/sunpy/issues) on our
+GitHub issue tracker.
```
Comment (wafels) [inline CONTRIBUTING.md]: Add "You will need to sign in to GitHub.com to report an issue.  If you are not already a member of Github.com, you will have to join.  Joining GitHub will also make it easier to report and track issues in the future".

If you do not want to join Github, then another way to report your issue is email the SunPy developers list sunpy-dev@googlegroups.com.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,20 +1,49 @@
...
+
+A good issue will provide instructions and a code sample on how someone else can
```
Comment (wafels) [inline CONTRIBUTING.md]: Issue reports should provide a short description of the issue.  If you are reporting a bug, please post a code sample so others can attempt to reproduce the error.  Please also post any error output generated by the bug.  For a good example of how to do this see +issue [#1191](https://github.com/sunpy/sunpy/issues/1191).

If you are making a feature request, please post as much information as possible regarding the feature you would like to see in SunPy.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -22,9 +51,9 @@ When you submit a pull request it can be reviewed and commented on by anyone.
 For the code to be accepted into SunPy it must meet the following criteria:
```
Comment (wafels) [inline CONTRIBUTING.md]: Add "Also, *everyone's* contributions gets reviewed, and no-one's code is accepted in to SunPy without being reviewed.  We do this to try to maintain a consistent coding style and high quality code that is easy to read and understand".

**Comment by Cadair** [comment]
Thanks for the comments @wafels.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,45 +1,94 @@
...
+git and GitHub, which are the tools SunPy uses to manage changes to the code and
+documentation.
+
+1. Get a [GitHub Account](https://github.com/signup/free)
+1. [Fork](https://help.github.com/articles/fork-a-repo) SunPy on GitHub
+1. Install the developer version of SunPy using git. 
+1. Make your changes and push to your fork.
+1. Create a [Pull Request](https://help.github.com/articles/using-pull-requests).
+
+If any of the above steps are alien to you check out our
```
Comment (wafels) [inline CONTRIBUTING.md]: If any of the above steps are unfamiliar

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -1,45 +1,94 @@
...
+
 For the code to be accepted into SunPy it must meet the following criteria:
 
 * Conform to code quality standards, primarily [PEP8](http://legacy.python.org/dev/peps/pep-0008/).
-* Have close to 100% unit test coverage.
+* Have close to 100% unit test coverage (see [testing](http://docs.sunpy.org/en/latest/dev.html#testing)).
 * Have API documentation generated by astropy's automodapi extenstion where appropriate.
-* Contain an entry in the CHANGELOG.md file.
+* Contain an entry in the [CHANGELOG.md](https://github.com/sunpy/sunpy/blob/master/CHANGELOG.md) file.
 * Be approved by at least two SunPy contributors.
```
Comment (wafels) [inline CONTRIBUTING.md]: Do you want to add in the fact we have a PEP8 checker now?

File: README.md [hunk 9]
```diff
@@ -7,33 +7,35 @@
...
- * [sqlalchemy](http://www.sqlalchemy.org/)
+    $ conda config --append channels conda-forge
+    $ conda install sunpy
 
-Next, use git to grab the latest version of SunPy:
 
-    git clone https://github.com/sunpy/sunpy.git
-    cd sunpy
-    python setup.py install
+If you want to develop SunPy you will need to install from git, the best way to
```
Comment (wafels) [inline README.md]: If you want to develop SunPy you will need to install using git.  The best way to

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,45 +1,94 @@
...
 
 
 ## Pull Requests to non-master Branches
 
 From time to time, large feature development work may occur in branches other than master, and all 
 maintained releases currently have a branch i.e. `0.6`.
-When making a PR to one of these branches, please put the name of the branch in square brackets at 
+When making a PR to one of these branches (not master), please put the name of the branch in square brackets at 
 the beginning of the PR, i.e. `[0.6] My bug fix`. This makes it easier to filter and review these PRs.
```
Comment (wafels) [inline CONTRIBUTING.md]: the beginning of the PR, i.e. `[0.6] My bug fix`. This makes it easier to filter and review these PRs on GitHub.

**Pull request merged**